browser/admin/src/integrator/WordBook.ts

```diff
@@ -160,7 +163,7 @@ class WordBook {
 
 		const wordNameDiv = document.createElement('div');
 		wordNameDiv.classList.add('list-item-content__name');
-		wordNameDiv.textContent = word;
+		wordNameDiv.textContent = _(word);
```

Comment:
I don't think it's necessary. Isn't it the word from the custom dictionary?

Reply:
yes, it's word so we can ignore it.